packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -1740,26 +1633,13 @@ Viewer.prototype.isDestroyed = function () {
  * removing the widget from layout.
  */
 Viewer.prototype.destroy = function () {
-  let i;
-  if (
-    defined(this.screenSpaceEventHandler) &&
-    !this.screenSpaceEventHandler.isDestroyed()
-  ) {
-    this.screenSpaceEventHandler.removeInputAction(
-      ScreenSpaceEventType.LEFT_CLICK
-    );
-    this.screenSpaceEventHandler.removeInputAction(
-      ScreenSpaceEventType.LEFT_DOUBLE_CLICK
-    );
-  }
-
   // Unsubscribe from data sources
   const dataSources = this.dataSources;
   const dataSourceLength = dataSources.length;
-  for (i = 0; i < dataSourceLength; i++) {
+  for (let i = 0; i < dataSourceLength; i++) {
     this._dataSourceRemoved(dataSources, dataSources.get(i));
   }
-  this._dataSourceRemoved(undefined, this._dataSourceDisplay.defaultDataSource);
+  this._dataSourceRemoved(undefined, this.dataSourceDisplay.defaultDataSource);
```

Comment:
This logic now lives in `CesiumWidget.destroy()` too, is it still needed here?

Reply:
Forgot this one